lib/buildsystems/autotools.rb

```diff
@@ -21,6 +22,12 @@ def self.build
         system 'autoreconf -fiv'
       end
     end
+    abort "configure script not found!".lightred if File.file?('configure')
```

Comment:
Will this fail the linter because of double quotes instead of single quotes?

Reply:
Apparently, not. :)

Reply:
I'll fix it in my next PR...